infrastructure/environments/dplplat01/sites.yaml

```diff
@@ -70,6 +70,7 @@ sites:
     description: "The library site for Aarhus"
     deploy_key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIFL+uMeEfsaHEzbNxOmBB8dX32OLo63CTomG8VZvuiN2"
     plan: webmaster
+    moduletest-dpl-cms-release: "2024.35.1"
```

Comment:
Since the moduletest-dpl-cms-release version equals the default version why not using only the default version and delete the moduletest-dpl-cms-release version until it differs in the future?

Reply:
you're right. Last week i had to do it a little fast, so that's why i did it like this